### ISSUE TITLE
opencv_world452d.dll missing

### ISSUE THREAD

**Issue by blobs0** [PtitGnou/AsciiVideoCPP#1]
hi, my computer can't find the dll "opencv_world452d" anyone have a solution? 
![image](https://user-images.githubusercontent.com/70648608/173807953-0b4fa019-9e94-4061-91ac-a8d28cecc38a.png)

**Comment by PtitGnou** [comment]
Hi, verify that opencv is installed on your computer and that it has been added to your windows system variable path. Here is the tutorial, https://answers.opencv.org/question/10454/where-can-i-find-dlls-for-opencv/ ; OpenCV can be installed from https://sourceforge.net/projects/opencvlibrary/ if it hasn't been done yet

**Comment by blobs0** [comment]
of course, I installed it but I still have this error, I've been having this problem for 1 hour x) 
![image](https://user-images.githubusercontent.com/70648608/173813420-48a4358e-919b-4895-8820-8abf65e26bf0.png)

**Comment by PtitGnou** [comment]
It shouldn't just be in the system's variables, but directly linked in the path variable :
<img width="428" alt="git1" src="https://user-images.githubusercontent.com/79204317/173896039-5930dabf-ad63-4d4f-8f09-902d67db50d4.png">
<img width="374" alt="git2" src="https://user-images.githubusercontent.com/79204317/173896054-2c9a7ed7-d919-46b0-9134-b2570cb09a10.png">
Try this and tell me if it works for you
